sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -1254,6 +1254,345 @@ public async Task StartRecognizeBusinessCardsCanParseMultipageForm(bool useStrea
+
+            var form = operation.Value.Single();
+
+            Assert.NotNull(form);
+
+            // The expected values are based on the values returned by the service, and not the actual
+            // values present in the invoice. We are not testing the service here, but the SDK.
+
+            Assert.AreEqual("prebuilt:invoice", form.FormType);
+            Assert.AreEqual(1, form.PageRange.FirstPageNumber);
+            Assert.AreEqual(1, form.PageRange.LastPageNumber);
+
+            Assert.NotNull(form.Fields);
+
+            Assert.True(form.Fields.ContainsKey("VendorName"));
+            Assert.True(form.Fields.ContainsKey("CustomerAddressRecipient"));
+            Assert.True(form.Fields.ContainsKey("VendorAddress"));
+            Assert.True(form.Fields.ContainsKey("CustomerAddress"));
+            Assert.True(form.Fields.ContainsKey("InvoiceId"));
+            Assert.True(form.Fields.ContainsKey("InvoiceDate"));
+            Assert.True(form.Fields.ContainsKey("DueDate"));
+            Assert.True(form.Fields.ContainsKey("InvoiceTotal"));
+            Assert.True(form.Fields.ContainsKey("CustomerName"));
+
+            Assert.AreEqual("Contoso", form.Fields["VendorName"].Value.AsString());
+            Assert.AreEqual("Microsoft", form.Fields["CustomerAddressRecipient"].Value.AsString());
+            Assert.AreEqual("1 Redmond way Suite 6000 Redmond, WA 99243", form.Fields["VendorAddress"].Value.AsString());
+            Assert.AreEqual("1020 Enterprise Way Sunnayvale, CA 87659", form.Fields["CustomerAddress"].Value.AsString());
+            Assert.AreEqual("Microsoft", form.Fields["CustomerName"].Value.AsString());
+
+            Assert.That(form.Fields["InvoiceTotal"].Value.AsFloat(), Is.EqualTo(56651.49f).Within(0.0001));
+
+            var invoiceDate = form.Fields["InvoiceDate"].Value.AsDate();
+            Assert.AreEqual(18, invoiceDate.Day);
+            Assert.AreEqual(6, invoiceDate.Month);
+            Assert.AreEqual(2017, invoiceDate.Year);
+
+            var dueDate = form.Fields["DueDate"].Value.AsDate();
+            Assert.AreEqual(24, dueDate.Day);
+            Assert.AreEqual(6, dueDate.Month);
+            Assert.AreEqual(2017, dueDate.Year);
+        }
+
+        [Test]
+        [TestCase(true)]
+        [TestCase(false)]
+        public async Task StartRecognizeInvoicesPopulatesExtractedTiff(bool useStream)
+        {
+            var client = CreateFormRecognizerClient();
+            RecognizeInvoicesOperation operation;
+
+            if (useStream)
+            {
+                using var stream = FormRecognizerTestEnvironment.CreateStream(TestFile.InvoiceLeTiff);
+                using (Recording.DisableRequestBodyRecording())
+                {
+                    operation = await client.StartRecognizeInvoicesAsync(stream);
+                }
+            }
+            else
+            {
+                var uri = FormRecognizerTestEnvironment.CreateUri(TestFile.InvoiceLeTiff);
```

Comment:
😄 I was wondering if this file was French or something because it's called "LeTiff". Then I realized it's "little endian" haha

Reply:
I had to look into the history of the file to see if it was a typo and then I google what the meaning was lol